File: A000201/A000201.cpp
// Lower Wythoff sequence (a Beatty sequence): a(n) = floor(n*phi), where phi = (1+sqrt(5))/2
// https://oeis.org/A000201

# include <iostream>
# include <cmath>
using namespace std;

int why_s(int n)
{
	return floor(n * ((1 + sqrt(5))/2));
}


int main()
{
	for (int i = 1; i <= 30; i++)
	{
		cout << why_s(i) << " ";
	}
	cout << "\n";
}

File: A000201/A000201.py
# Lower Wythoff sequence (a Beatty sequence): a(n) = floor(n*phi), where phi = (1+sqrt(5))/2
# https://oeis.org/A000201

import math

# Lower Whythoff sequence function
def why_s(n):
    return math.floor(n * ((1 + math.sqrt(5))/2))

# Printing first 30 numbers in the A000201.
for number in range(1, 31):
    print(why_s(number), end=" ")
print()
